navbar/navbar.css

```diff
@@ -4,7 +4,7 @@
     position: sticky;
     top: 0;
     left: 0;
-    z-index: 2;
+    z-index: 4;
```

Comment:
Any particular reason to give z-index as 4 as the value of z-index doesn't matter as long as there is only on element.

Reply:
because I already used 2 somewhere in the project and I want nav always on top so I used